engine/schema/mysql.sql

```diff
@@ -203,16 +185,16 @@ CREATE TABLE `prefix_queue` (
 
 -- activity stream
 CREATE TABLE `prefix_river` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
   `type` varchar(8) NOT NULL,
   `subtype` varchar(32) NOT NULL,
   `action_type` varchar(32) NOT NULL,
   `access_id` int(11) NOT NULL,
```

Comment:
that kind of scares me a little bit, honestly. Huge migration. Major breaking change -- how would the existing data models overlap, if at all?